backend/internal/data/shutter_explorer.sql.go

```diff
@@ -229,6 +229,38 @@ func (q *Queries) QueryLatestTXsWhichArentIncluded(ctx context.Context, limit in
 	return items, nil
 }
 
+const queryTotalRegisteredValidators = `-- name: QueryTotalRegisteredValidators :many
+SELECT COUNT(DISTINCT validator_index) AS total_registered_validators
+FROM validator_registration_message
+WHERE is_registeration = TRUE 
+AND validity = 'valid'
```

Comment:
what does the validity part mean? What is being checked that validity is true?

Reply:
validity basically tells if message is valid or if something is wrong with it. Its a db type